registrations/serializers.py

```diff
@@ -1551,7 +1552,7 @@ class RegistrationSignupsExportSerializer(serializers.Serializer):
 
 
 class PriceGroupSerializer(TranslatedModelSerializer, CreatedModifiedBaseSerializer):
-    def _description_is_valid(self, data, validated_data):
+    def _any_required_description_field_has_a_value(self, data, validated_data):
```

Comment:
I find the "required" part of the name of this method a bit confusing. Sure this is used to check the required field but it is not in the scope of this function.
I suggest naming this function as: `_has_any_description_field_a_value` or `_has_value_in_any_description_field` 🧐

Reply:
There are more supported languages for the description field than the three that are checked in the method - this method checks only those that are "required" by the payments integration (at least one of "fi", "sv" or "en" needs to be given as those are supported in Talpa and in the LE UI). Thus the "required" in the method name 😎

Reply:
I renamed the function to reflect the fact that it is actually ensuring required translations: `_any_required_description_translation_exists`. Also clarified this in the function's comment.